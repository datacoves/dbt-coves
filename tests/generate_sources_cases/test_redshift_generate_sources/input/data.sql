INSERT INTO tests.test_model VALUES (
    1,
    1,
    1,
    1.1,
    1.1,
    'a',
    'a',
    '2020-01-01',
    '2020-01-01 00:00:00',
    TRUE,
    JSON_PARSE('{"name":"abc","value": 2}')
);
