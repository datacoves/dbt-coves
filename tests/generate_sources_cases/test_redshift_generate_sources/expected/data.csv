test_smallint,test_integer,test_bigint,test_decimal,test_numeric,test_char,test_varchar,test_date,test_timestamp,test_boolean,test_json
1,1,1,1,1,a,a,2020-01-01,2020-01-01,True,"{""name"":""abc"",""value"":2}"
